src/components/ClusterManager.js

```diff
@@ -260,7 +260,7 @@ export default class ClusterManager extends PureComponent {
           })
       }
     }
-    const totalCost = _.sum(_.map(clusterCost, clusters))
+    const totalCost = _.sum(_.map(clusterCost, clusters)) + _.sum(_.map(persistentDiskCost, persistentDisks))
```

Comment:
Opt, TOL: Could we fold the sums together? Or is it worth wrapping in a sum call instead of a +?

Reply:
I thought about it, this way felt slightly clearer to me.